### DISCUSSION TITLE
add support for paging and idle procedures to srsran gNB and srsUE ZMQ

### DISCUSSION THREAD

**Discussion by mohamedalyk** [srsran/srsRAN_Project#508] [Ideas & Suggestions]
we wish to see idle procedures (paging, service request, periodic tracking area update) in ZMQ gNB and 5G  srsUE like the LTE counterpart.

Thanks

**Comment by pgawlowicz** [comment]
while we are constantly developing the srsRAN-Project gNB, the development of the srsUE (and its 5G features) is currently of a low-priority and the requested features are not going to be implemented soon.
